plugins/jira/skills/catch-me-up/scripts/split_batches.py

```
@@ -0,0 +1,92 @@
+    }
+
+    groups = defaultdict(list)
+    for event in events:
+        groups[event["issue"]["key"]].append(event)
+
+    max_batch_size = batch_size * 3
+
+    batches: list[list[dict]] = []
+    current_batch: list[dict] = []
+
+    for issue_key, issue_events in groups.items():
```

Ruff (0.15.10)

[warning] Loop control variable `issue_key` not used within loop body

Rename unused `issue_key` to `_issue_key`

(B007)